asmcli/commands/experimental/install.sh

```diff
@@ -7,31 +7,12 @@ x_install_subcommand() {
 
   x_validate_dependencies
   x_configure_package
+  post_process_istio_yamls
   x_install
 }
 
 x_install() {
-  local PROJECT_ID; PROJECT_ID="$(context_get-option "PROJECT_ID")"
-  local CLUSTER_LOCATION; CLUSTER_LOCATION="$(context_get-option "CLUSTER_LOCATION")"
-  local CLUSTER_NAME; CLUSTER_NAME="$(context_get-option "CLUSTER_NAME")"
-  local USE_MANAGED_CNI; USE_MANAGED_CNI="$(context_get-option "USE_MANAGED_CNI")"
-
-  if [[ "${USE_MANAGED_CNI}" -eq 0 ]]; then
-    install_mananged_cni_static
-  fi
-  apply_kube_yamls
-
-  # `kubectl wait` for non-existent resources will return error directly so we wait in loop
-  info "Wait for the controlplanerevisions CRD to be installed by AFC. This should take a few minutes if cluster is newly registered."
-  for i in {1..10}; do
-    if kubectl wait --for condition=established --timeout=10s crd/controlplanerevisions.mesh.cloud.google.com 2>/dev/null; then
-      break
-    fi
-    sleep 10
-  done
-
-  install_control_plane_revisions
-
+  install_control_plane
```

Comment:
With the change here, it seems that x_install() will no longer work for the managed control plane. I think x_install should support both the in-cluster control plane and the managed control plane.

Reply:
don't think so. the install_control_plane function branches out into managed and regular control plane installation?